source/config/configSpec.py

```diff
@@ -85,6 +85,7 @@
 	reviewRoutingMovesSystemCaret = featureFlag(\
 		optionsEnum="ReviewRoutingMovesSystemCaretFlag", behaviorOfDefault="NEVER")
 	readByParagraph = boolean(default=false)
+	paragraphStartMarker = option("", " ", "¶", default="")
```

Comment:
can you please make this an int to correspond to the index of the option?

Reply:
Can you provide more details about this, please?